edit-post/store/test/reducer.js

```diff
@@ -35,6 +35,24 @@ describe( 'state', () => {
 			expect( state.activeGeneralSidebar ).toBe( 'edit-post/document' );
 		} );
 
+		it( 'should save activeGeneralSidebar default value when serializing if the value was edit-post/block', () => {
+			const state = preferences( {
+				activeGeneralSidebar: 'edit-post/block',
```

Comment:
I personally think we should not persist the `activeGeneralSidebar` in all use-cases? cc @jasmussen

Reply:
Not sure what the consequences are. The specific use case that should be solved is that when you write a new post, you should always see the Document sidebar, until you either manualy pick the block tab or click a block. Otherwise the auto-select-block-tab behavior becomes sort of unpredictable, as per #6377

Reply:
My thinking is that since we have auto-open behavior for the tabs we don't need to persist them anymore since they'll be visible when necessary.

Reply:
Perhaps, but do you disagree with the behavior described in #6377 being a problem? 

To me, clicking "add new post" _implicitly_ means that I've deselected all blocks, which should restore the document tab no?

Reply:
Hi @youknowriad, I think the tabs were being persisted because of the plugins. If we have a plugin tab open, we should keep it open when we reload.

Reply:
Not at all. I'm just trying to think if the fix couldn't be to just remove the persistence.